tdrs-backend/test

```diff
@@ -0,0 +1 @@
+test
```

Comment:
This should also be nixed.

Reply:
Can we add these to the gitignore for now until we can address it properly